### PR TITLE
Multiple statistics annotations on one plot

### PR DESCRIPTION
**Pull request by danholdaway**
## Description

Allow the adding of statistics annotation of multiple plots. E.g. from the tests:

gsi_omb_jedi_omb_density_amsua_n19_brightnessTemperature_3

## Dependencies

Not technically a dependency since eva is based on a specific emcpy commit but the changes should probably be accepted.
- [ ] https://github.com/NOAA-EMC/emcpy/pull/121

## Impact

N/A

### CODE REVIEW

**Review by CoryMartin-NOAA — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -39,10 +39,10 @@
         'cartopy>=0.18.0',
         'scikit-learn>=1.0.2',
         'xarray>=0.11.3',
-        #  'seaborn==0.12',
         'emcpy @ git+https://github.com/NOAA-EMC/' +
-        'emcpy@dcdc868e7f48bb1ff55507f34c0548742af8bec1#egg=emcpy',
-        # 'seaborn==0.12',
+        'emcpy@668fab9dcfdd8885cd2cd971478368782cc5db6f#egg=emcpy',
```
Comment (CoryMartin-NOAA) [inline setup.py]: is this the right hash?

File: src/eva/utilities/stats.py [hunk 2]
```diff
@@ -1,263 +1,133 @@
...
+            if stats_variable in ['n']:
+                stat_value = len(field_data)
+            elif stats_variable in ['min', 'max', 'mean', 'median', 'std', 'var']:
+                stat_value = eval(f'np.nan{stats_variable}(field_data)')
+                stat_value = eval(f'np.round(stat_value, {digits})')
+            else:
+                logger.abort(f'In stats_helper the statistic {stats_variable} is not supported.')
+
+            stats_string = stats_string + f'{stats_variable} = ' + \
+                           format_dict[stats_variable].format(stat_value)
```
Comment (CoryMartin-NOAA) [inline src/eva/utilities/stats.py]: PEP8

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]

File: src/eva/utilities/stats.py (hunk 2 above)
Comment (danholdaway) [reply src/eva/utilities/stats.py]: Done.

File: setup.py (hunk 1 above)
Comment (danholdaway) [reply setup.py]: Now the PR is merged I've updated the hash to reflect the squash.

**Review by CoryMartin-NOAA — APPROVED** [review]
Thanks Dan!

**Pull request merged**